integration-tests/s3_test.go

```diff
@@ -95,6 +95,7 @@ var _ = Describe("S3", Label("s3"), func() {
 				"enable_versioning":     true,
 				"region":                "eu-west-1",
 				"acl":                   "public-read",
+				"boc_object_ownership":  "BucketOwnerPreferred",
```

Comment:
thoughts: you could check the default value in line 75 (provision)

Reply:
Good catch - this has been added, thanks